Configuration/StandardSequences/python/Reconstruction_cff.py

```diff
@@ -65,12 +66,12 @@
 phase2_timing_layer.toReplaceWith(localreco_HcalNZS,_phase2_timing_layer_localreco_HcalNZS)
 
 _ctpps_2016_localreco = localreco.copy()
-_ctpps_2016_localreco += totemRPLocalReconstruction
+_ctpps_2016_localreco += totemRPLocalReconstruction+ctppsDiamondLocalReconstruction
```

Comment:
it's fine to leave as is.
The change makes sense in the local context, but then in the CTPPS general reco sense, "local" modifier isn't really applicable because all of the CTPPS reco is done using just the CTPPS detectors.